packages/block-library/src/list/block.json

```diff
@@ -12,6 +12,22 @@
 			"selector": "ol,ul",
 			"multiline": "li",
 			"default": ""
+		},
+		"start": {
+			"type": "string",
+			"default": ""
+		},
+		"reversed": {
+			"type": "boolean",
+			"default": false
+		},
+		"type": {
+			"type": "string",
+			"default": "1"
+		},
+		"description": {
```

Comment:
What is "description"?

Reply:
This information could be derived from "type"?

Reply:
Done